### PR TITLE
Burnsbert patch 1

### PR DESCRIPTION
**Pull request by burnsbert**
I'd like to request the tagtacular.js library be added to the CDN. Thanks.

https://github.com/burnsbert/tagtacular

### CODE REVIEW

File: files/tagtacular/update.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "tagtacular",
+  "repo": "burnsbert/tagtacular",
+  "files": {
+    "include": ["tagtacular.js", "tagtacular.min.js", "tagtacular.css"]
```
Comment (jimaek) [inline files/tagtacular/update.json]: These files dont exist here https://github.com/burnsbert/tagtacular/tree/1.0.1
The bot will fail

**Comment by burnsbert** [comment]
Sorry for the earlier mistake. I think I corrected it.

File: files/tagtacular/update.json (hunk 1 above)
Comment (burnsbert) [inline files/tagtacular/update.json]: Sorry, I should have included the path on those files in update.json. I
think I corrected the issue in the commit I just made.

Eric

**Comment by jimaek** [comment]
Do you want the files to retain their directory structure?
Right now they wont but if you add `./` at the begging of each file then they will.
Also remove the unminified file, the min.js is enough for production.

**Comment by burnsbert** [comment]
Thank you very much for your help. I made and committed the suggested changes.

**Comment by burnsbert** [comment]
Thank you very much for your help. I made and committed the suggested
changes.

Eric

**Comment by jimaek** [comment]
Last change. Rename it to `jquery.tagtacular`

**Comment by burnsbert** [comment]
Done.

**Comment by burnsbert** [comment]
Done.

Thanks,
Eric

**Comment by jimaek** [comment]
@jsdelivrbot squash and merge

**Pull request closed without merging**